pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,47 @@
+package backuphelpers
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+)
+
+type BackupVar interface {
+	ArgString() string
+}
+
+type BackupConfig struct {
+	enabled  bool
+	backupCR backupv1alpha1.EtcdBackupSpec
+	mux      sync.Mutex
+}
+
+func NewDisabledBackupConfig(backupCR backupv1alpha1.EtcdBackupSpec, enabled bool) *BackupConfig {
```

Comment:
let's leave this without parameters, the testcases seem to directly use the struct anyway
```suggestion
func NewDisabledBackupConfig() *BackupConfig {
```

Reply:
i need to pass the CR as an param for testing